### PR TITLE
chore: bump envd dependency to v0.2.0

### PR DESCRIPTION
**Pull request by cutecutecat**
- update envd dependency
- update cli/v2 dependency
- fix CI echo clerical error

Signed-off-by: cutecutecat <starkind1997@gmail.com>

### CODE REVIEW

**Comment by gaocegege** [comment]
Please assign me to review if it is ready.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -1,16 +1,18 @@
 name: release
-
 on:
   push:
     tags:
       - 'v*'
-  pull_request:
+  pull_request_target:
```
Comment (kemingy) [inline .github/workflows/release.yml]: This one is the real dangerous method 🤣 

It will pass the write permission to fork repos.

Comment (cutecutecat) [reply .github/workflows/release.yml]: Sorry, this is a test of several ways to solve that problem, after nearly a hour, I concluded that:
Leaving it as it is is probably the best thing to do.

**Comment by cutecutecat** [comment]
> Please assign me to review if it is ready.

@gaocegege

**Review by gaocegege — APPROVED** [review]

**Pull request merged**